slang-kotlin/src/main/java/com/sonarsource/slang/kotlin/KotlinRulesDefinition.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * SonarSource SLang
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonarsource.slang.kotlin;
+
+import com.sonarsource.slang.checks.CommonCheckList;
+import org.sonar.api.server.rule.RulesDefinition;
+import org.sonarsource.analyzer.commons.RuleMetadataLoader;
+
+public class KotlinRulesDefinition implements RulesDefinition {
+
+  private static final String RESOURCE_FOLDER = "org/sonar/l10n/kotlin/rules/kotlin";
+
```

Comment:
Did you mean to put this in a 'org/sonar/l10n/slang/rules/kotlin' folder instead? I don't see the need to have twice the 'kotlin' in the path

Reply:
It used to be a SonaQube requirement a very long time ago: one "kotlin" represents the plugin key and other one is for the language key. I think we should keep it only if we insist on being very consistent with other plugins. As this is not a requirement anymore and as this path looks really strange, I would rather change it to something simpler.

Reply:
Oh ok. Thx for info.
I'd be fine with a simpler path if it's not required anymore :)
Let's discuss this on thursday